### PR TITLE
Onomatopoeias

### PR DESCRIPTION
**Pull request by IrishKnight0**
There are so many of them so tell me what to revert.

### CODE REVIEW

File: script/Arcueid/Day 4/04_00_ARC04_6.txt [hunk 1]
```diff
@@ -569,7 +569,7 @@ And then,
 [sha:91b18c48b731adb279d48df631bbc1d027604b0c]{
 -- Page 406, Offset 6686. Glued. Mods: @n.
 -- と弾むような足取りでベッドまで移動すると、
- with a bounce in her step,
+ with a %{i}bounce%{/i} in her step,
```
Comment (rschlaikjer) [inline script/Arcueid/Day 4/04_00_ARC04_6.txt:572]: I wouldn't consider this an onomatopoeia

File: script/Arcueid/Day 4/04_00_ARC04_6_2D.txt [hunk 2]
```diff
@@ -41,7 +41,7 @@ My antibodies can render secret arts like Runes and Kabbalah useless, yet someho
 [sha:b362b3538cab1cf4a3f2e755bd1255a823859c06]{
 -- Page 566, Offset 7092.
 -- 「いえ、それだってあそこまでわたしを“殺して”おく事はできない。
-"No, even those wouldn't be able to go so far as to outright 'kill' me.
+"No, even those wouldn't be able to go so far as to outright %{i}kill%{/i} me.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 4/04_00_ARC04_6_2D.txt:44]: I'd prefer single quotes here, italics are more like emphasis but quotes indicate that the meaning is different to usual imo.

File: script/Arcueid/Day 5/05_00_ARC05_3.txt [hunk 3]
```diff
@@ -6,7 +6,7 @@
 [sha:bcac3d7504b37322c7785e42c0b22a5b7f737149]{
 -- Page 10005, Offset 9133.
 -- 志貴が一緒に戦ってくれるのは助かるしね。あなたならヴローヴなんて問題なく一撃で“殺”せるんだから」
-I'm glad that you've decided to fight by my side. After all, we can easily 'kill' Vlov in one blow with you around."
+I'm glad that you've decided to fight by my side. After all, we can easily %{i}kill%{/i} Vlov in one blow with you around."
```
Comment (rschlaikjer) [inline script/Arcueid/Day 5/05_00_ARC05_3.txt:9]: Quotes for same reason as above

File: script/Arcueid/Day 5/05_00_ARC05_9.txt [hunk 4]
```diff
@@ -535,7 +535,7 @@ That leaves more than enough. I can already see them―――
 -- Page 591, Offset 10210.
 -- あの怪物を“殺す”、死の線が視えている。
 // あの怪物を"殺す"、死の線が視えている。
-the lines of death that will 'kill' this monster.
+the lines of death that will %{i}kill%{/i} this monster.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 5/05_00_ARC05_9.txt:538]: Ditto

File: script/Arcueid/Day 1/01_00_ARC01_8.txt [hunk 5]
```diff
@@ -647,7 +647,7 @@ Hisui nods, answering yes to my question, and puts her hand on the doorknob.
 [sha:afaaaca174c7372614f0ac7bdd280e7d7134d186]{
 -- Page 498, Offset 2419.
 -- きぃ、とかすかな軋み音をたててドアが開かれる。
-The door swings open with a faint creak.
+The door swings open with a faint %{i}creak%{/i}.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 1/01_00_ARC01_8.txt:650]: Probably drop

File: script/Arcueid/Day 3/03_00_ARC03_1_2.txt [hunk 6]
```diff
@@ -156,7 +156,7 @@ As her brother, it would have made sense for me to talk to her first...
 [sha:3c569ca66f0a817b900608cecff4d05312053f42]{
 -- Page 55, Offset 4084.
 -- ふん、と鼻をならす秋葉。
-With a hmph, Akiha replies.
+With a %{i}hmph%{/i}, Akiha replies.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 3/03_00_ARC03_1_2.txt:159]: Probably drop

File: script/Arcueid/Day 2/02_00_ARC02_4_3.txt [hunk 7]
```diff
@@ -12,7 +12,7 @@ It would be a shame to lose this feeling of satisfaction by walking around since
 [sha:36f4f0714d294e731a5a360b391d7118a6d07819]{
 -- Page 10017, Offset 3386.
 -- そうして、どたーん！　とベッドに体を投げ出す遠野志貴なのであった。
-Then, with a thump, I throw myself on the bed.
+Then, with a %{i}thump%{/i}, I throw myself on the bed.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 2/02_00_ARC02_4_3.txt:15]: Probably drop

File: script/Arcueid/Day 3/03_00_ARC03_3.txt [hunk 8]
```diff
@@ -128,7 +128,7 @@ Arihiko, for his part, has plopped himself down on someone's desk, happily chatt
 [sha:4377a2644bba0d7c23ed3f1cc80a3e8ad0575130]{
 -- Page 136, Offset 4294.
 -- なんだかどっと疲れてしまって、ため息まじりに椅子に座った。
-I sit down on my chair with a sigh, feeling somewhat tired all of a sudden.
+I sit down on my chair with a %{i}sigh%{/i}, feeling somewhat tired all of a sudden.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 3/03_00_ARC03_3.txt:131]: Probably drop

File: script/Arcueid/Day 4/04_00_ARC04_10.txt [hunk 9]
```diff
@@ -680,7 +680,7 @@ And yet, just before that―――
 [sha:ae25bc1d05c4843655386cca9641c8608f826a2d]{
 -- Page 847, Offset 7913.
 -- ざくん、という音とともに、俺の体は後ろに引っ張られていた。
-My body is yanked backwards, and at the same time a thud reverberates in the hallway.
+My body is yanked backwards, and at the same time a %{i}thud%{/i} reverberates in the hallway.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 4/04_00_ARC04_10.txt:683]: Probably drop

File: script/Arcueid/Day 4/04_00_ARC04_10.txt [hunk 10]
```diff
@@ -826,7 +826,7 @@ After expelling a breath like exhaust from an engine, the man reaches under his
 [sha:b7957b1d3029da359b444a7db6779e2991bb9d9a]{
 -- Page 859, Offset 7942.
 -- カチャリ、とかすかな金属音が聞こえた。
-I hear a faint metallic click.
+I hear a faint metallic %{i}click%{/i}.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 4/04_00_ARC04_10.txt:829]: Probably drop

File: script/Arcueid/Day 4/04_00_ARC04_10.txt [hunk 11]
```diff
@@ -1118,7 +1118,7 @@ Not the two dogs that attacked me, not the flames that covered the corridor, and
 [sha:7ef9ad8a5c24e6c4e321ba42244eeb49b3131a5a]{
 -- Page 879, Offset 8000.
 -- どん、とアルクェイドが寄りかかってきた。
-Arcueid collapses against me with a thud.
+Arcueid collapses against me with a %{i}thud%{/i}.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 4/04_00_ARC04_10.txt:1121]: Probably drop

File: script/Arcueid/Day 4/04_00_ARC04_3.txt [hunk 12]
```diff
@@ -722,7 +722,7 @@ I put my hands on the wall and try to stand up, but it was no use.
 [sha:3db9f2611740b25647ad1a14f0f17f34d188adf7]{
 -- Page 143, Offset 5954.
 -- 立ち上がったとたん膝から力を失って、ドスンと尻餅をつく。
-As soon as I stand up, my knees lose strength and I fall on my backside with a thud.
+As soon as I stand up, my knees lose strength and I fall on my backside with a %{i}thud%{/i}.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 4/04_00_ARC04_3.txt:725]: Probably drop

File: script/Arcueid/Day 4/04_00_ARC04_3.txt [hunk 13]
```diff
@@ -957,7 +957,7 @@ Eighteen hours later, I finally caught you."
 [sha:df5bbaa21cd86d7ae776f6a20b18197508e1760e]{
 -- Page 156, Offset 6001.
 -- かつん、と更に一歩、女は路地裏に入ってくる。
-With a clunk, the woman takes another step into the alleyway.
+With a %{i}clunk%{/i}, the woman takes another step into the alleyway.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 4/04_00_ARC04_3.txt:960]: Probably drop

File: script/Arcueid/Day 4/04_00_ARC04_6_2D.txt [hunk 14]
```diff
@@ -266,7 +266,7 @@ If that's the case, then of course you'd be able to kill me."
 [sha:795984a59b532fae5152dd000b01c880010c6c6c]{
 -- Page 584, Offset 7137.
 -- ふん、と不機嫌そうに鼻を鳴らして、アルクェイドは俺から視線を逸らした。
-With a huff, Arcueid pouts and averts her gaze from me.
+With a %{i}huff%{/i}, Arcueid pouts and averts her gaze from me.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 4/04_00_ARC04_6_2D.txt:269]: If using italics, I'd split the sentence
> %{i}Hmph%{/i}. Arcueid pouts and...

File: script/Arcueid/Day 4/04_00_ARC04_8B.txt [hunk 15]
```diff
@@ -296,7 +296,7 @@ How many people's blood do you think I have sucked so far?"
 [sha:369e229cc97c98b2f470226e1885cf28376d7ba5]{
 -- Page 683, Offset 7389.
 -- タン、と軽やかにベッドから跳ねて、アルクェイドは窓の近くまで歩いていく。
-With a tap, Arcueid springs lightly from the bed and walks over to the window.
+With a %{i}tap%{/i}, Arcueid springs lightly from the bed and walks over to the window.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 4/04_00_ARC04_8B.txt:299]: Probably drop

File: script/Arcueid/Day 4/04_00_ARC04_9_1.txt [hunk 16]
```diff
@@ -812,7 +812,7 @@ The dog shakes its body and forcefully positions itself on top of me.
 [sha:1170dba77d68b073430f733595a79030556bd815]{
 -- Page 771, Offset 7663.
 -- だん、と床に倒れこむ。
-I collapse to the floor with a thud.
+I collapse to the floor with a %{i}thud%{/i}.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 4/04_00_ARC04_9_1.txt:815]: Probably drop

File: script/Arcueid/Day 4/04_00_ARC04_9_1.txt [hunk 17]
```diff
@@ -707,7 +707,7 @@ I thrust the knife into its open mouth.
 [sha:e6a534464ab7b68ba5f63d16dc83dce0f3c3ef4b]{
 -- Page 763, Offset 7642.
 -- ずぶり、と肉に刃物が埋まっていく。この犬は外皮こそ硬いが、その中身は軟らかい。文字通り、骨も肉も腐っている。
-With a stab, the blade burrows itself into its flesh. The dog's outer skin is hard, but its insides are soft. Its bones and flesh are both literally rotten.
+With a %{i}stab%{/i}, the blade burrows itself into its flesh. The dog's outer skin is hard, but its insides are soft. Its bones and flesh are both literally rotten.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 4/04_00_ARC04_9_1.txt:710]: Probably drop

File: script/Arcueid/Day 4/04_00_ARC04_9_1.txt [hunk 18]
```diff
@@ -587,7 +587,7 @@ The moment I realize it's coming.
 [sha:ea731c040ad55b2c9b46a549ae9681b29b853801]{
 -- Page 755, Offset 7618.
 -- 黒犬の牙は、ガギリ、と音をたてて俺の喉に食いついた。
-The dog's fangs sink into my throat with a wet squelch.
+The dog's fangs sink into my throat with a wet %{i}squelch%{/i}.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 4/04_00_ARC04_9_1.txt:590]: Probably drop

File: script/Arcueid/Day 5/05_00_ARC05_8.txt [hunk 19]
```diff
@@ -543,7 +543,7 @@ Arcueid comes to a sudden halt and sticks her hands out in a protective gesture.
 -- Page 552, Offset 10084.
 -- 吹き<荒|すさ>び、金切り声をあげ、街を覆う絶対零度。
 // 吹き<荒|すさ>び、金切り声をあげ、街を覆う絶対零度。
-The wind blows fiercely, and with a piercing howl envelops the city in absolute zero.
+The wind blows fiercely, and with a piercing %{i}howl%{/i} envelops the city in absolute zero.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 5/05_00_ARC05_8.txt:546]: Probably drop

File: script/Arcueid/Day 5/05_00_ARC05_1B.txt [hunk 20]
```diff
@@ -984,7 +984,7 @@ I reach out for her white clothes.
 [sha:39331686f5c127692f74f79241feeb65324f0cca]{
 -- Page 118, Offset 8869.
 -- アルクェイドはごろん、と音をたてて、ベッドの上で転がった。
-Arcueid rolls over on the bed with a flop.
+Arcueid rolls over on the bed with a %{i}flop%{/i}.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 5/05_00_ARC05_1B.txt:987]: Probably drop

File: script/Arcueid/Day 5/05_00_ARC05_1.txt [hunk 21]
```diff
@@ -279,7 +279,7 @@ Neither mine nor Arcueid's name is on the list.
 [sha:9d524735ed083e7f37bf36909764825721153778]{
 -- Page 24, Offset 8646.
 -- ぱちん、とテレビの電源を切った。
-With a click, I turn off the TV.
+With a %{i}click%{/i}, I turn off the TV.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 5/05_00_ARC05_1.txt:282]: Probably drop

File: script/Arcueid/Day 5/05_00_ARC05_1.txt [hunk 22]
```diff
@@ -71,7 +71,7 @@ After doing what I could, I should finally be able to relax.
 [sha:7cf739aa2eebbbfc57ec4999f7a27d86a0d01cbc]{
 -- Page 7, Offset 8605.
 -- 座ろうとした途中、膝から力が抜けて、どすん、と床に倒れこんだ。
-But on my way to go sit down, my knees give out and I fall to the floor with a thud.
+But on my way to go sit down, my knees give out and I fall to the floor with a %{i}thud%{/i}.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 5/05_00_ARC05_1.txt:74]: Probably drop

**Review by rschlaikjer — COMMENTED** [review]
My opinion would be to only use italic only for standalone onomatopoeia as a general rule, obviously exceptions can exist. But generally -

> _Clunk_. The gate swings open // Standalone, italics
> The gate clunks and swings open. // Integrated, no italics

File: script/Arcueid/Day 1/01_00_ARC01_5.txt [hunk 23]
```diff
@@ -66,7 +66,7 @@ I open the heavy door and enter the premises, passing through the stone gate.
 -- Page 248, Offset 1797.
 -- 背後で、カシュ、と小気味よい音をたて、扉は固く<施錠|ロック>された。
 // 背後で、カシュ、と小気味よい音をたて、扉は固く<施錠|ロック>された。
-The door locks tightly behind me with a satisfying clunk.
+The door locks tightly behind me with a satisfying %{i}clunk%{/i}.
```
Comment (IrishKnight0) [inline script/Arcueid/Day 1/01_00_ARC01_5.txt:69]: Why not revert this btw?

File: script/Arcueid/Day 3/03_00_ARC03_2B.txt [hunk 24]
```diff
@@ -277,7 +277,7 @@ It's like that. Okay, here we go."
 [sha:38b27ebe287f76392641a5d7f5a225ecf91dd58d]{
 -- Page 104, Offset 4219.
 -- パチン、と音がして10センチほどの刃が飛び出した。
-With a snap, a blade that is about ten centimeters long springs out.
+With a %{i}snap%{/i}, a blade that is about ten centimeters long springs out.
```
Comment (IrishKnight0) [inline script/Arcueid/Day 3/03_00_ARC03_2B.txt:280]: This too why not revert it?

File: script/Arcueid/Day 1/01_00_ARC01_5.txt (hunk 23 above)
Comment (rschlaikjer) [reply script/Arcueid/Day 1/01_00_ARC01_5.txt:69]: I just missed these ones

**Pull request merged**